lib/mongoid/history/tracker.rb

```diff
@@ -179,6 +179,13 @@ def create_standalone
 
       def create_on_parent
         name = association_chain.last['name']
+
+        #
+        # TODO: modifier
+        # klass = trackable_parent.class.relation_class_of(name)
+        # klass.history_trackable_options[:modifier_field] if klass.respond_to?(:history_trackable_options)
+        #
```

Comment:
Please delete this comment, it's just cruft.

Reply:
Removed.